client/src/api/shows.js

```diff
@@ -28,5 +28,11 @@ export async function getShowDetails(showId) {
     throw new Error(response.statusText);
   }
   const showDetails = await response.json();
+  showDetails.actors.map((actor) => {
+    if (actor.picture === 'https://image.tmdb.org/t/p/w92null') {
```

Comment:
is that a common `src` in your API when they don't have a picture? Might want to consider using your `DefaultPicture` here instead of doing two checks and two replacements (`client/src/components/ActorList.js` line 97)

Reply:
yeah good find. I added the default picture after the fix in the map 😅